include/nighthawk/adaptive_load/adaptive_load_controller.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "envoy/common/time.h"
+
+#include "api/adaptive_load/adaptive_load.pb.h"
+#include "api/client/service.grpc.pb.h"
+
+namespace Nighthawk {
+namespace AdaptiveLoad {
+
+// Performs an adaptive load session defined by |spec| using the Nighthawk Service at
```

Comment:
Consider using the [docstring](https://www.doxygen.nl/manual/docblocks.html) format where applicable; we use that in other places too, and I think it helps tooling and IDEs leverage it.  [Example](https://github.com/envoyproxy/nighthawk/blob/c57811f3626b53f997f0241aaa482366dfe790c0/include/nighthawk/client/options.h#L23).

Reply:
Done